lighthouse-core/audits/non-composited-animations.js

```diff
@@ -94,9 +103,11 @@ class NonCompositedAnimations extends Audit {
 
       const animations = element.animations || [];
       const animationReasons = new Map();
-      for (const {name, failureReasonsMask} of animations) {
+      for (const {name, failureReasonsMask, unsupportedProperties} of animations) {
         if (!failureReasonsMask) continue;
-        for (const failureReason of getActionableFailureReasons(failureReasonsMask)) {
+        const failureReasons =
+          getActionableFailureReasons(failureReasonsMask, unsupportedProperties || []);
+        for (const failureReason of failureReasons) {
           if (name) {
             hasDisplayNames = true;
```

Comment:
wasnt sure which name this was referring to. how about `shouldAddAnimationNamesColumn`? super wordy but it works.